source/docs/prismic/embed-fields.md

```diff
@@ -0,0 +1,231 @@
+
+```js
+const AlbumLoader = (PrismicLoader) => {
+  const { EmbedTransformer } = PrismicLoader.transformers;
+
+  const contentTransformOptions = {
+    fieldTransformers: {
+      // ... other transformers
+      cover: new EmbedTransformer(),
+    },
+  };
+
+  return {
+    loadByUID: (uid) => {
+      return PrismicLoader.loadByUID("album", uid, contentTransformOptions);
+    },
+  };
+};
+```
+
+### Adding an Embed Field in your client-side
+
+We have added a `PrismicEmbed` component to handle the embed response in you client side.
+
+To simplify the usage between the component and the backend data we created a `PrismicEmbedFragment` which contains the minimum required data for the component, for example:
+
+```graphql
+#import "theme/modules/prismic/PrismicEmbed/PrismicEmbedFragment"
+
+fragment AlbumFragment on Album {
+  title
+  cover {
+    ...PrismicEmbedFragment
+  }
+}
+```
+
+The fragment will expose the `html`, `provider` and `type` fields in the [`oEmbedContent`](https://gitlab.com/front-commerce/front-commerce-prismic/-/blob/main/prismic/server/modules/prismic/core/schema.gql) type.
+
+```jsx
+import PrismicEmbed from "theme/modules/prismic/PrismicEmbed";
+
+const Album = (props) => {
+  return (
+    <div>
+      <h1>{props.title}</h1>
+      <PrismicEmbed
+        html={props.cover.html}
+        type={props.cover.type}
+        provider={props.cover.provider}
+      />
+    </div>
+  );
+};
+```
+
+You can override the default components for each embed type in your own theme
+
+| Type    | Component                                                |
+| ------- | -------------------------------------------------------- |
+| `photo` | `theme/modules/prismic/PrismicEmbed/Components/Photo.js` |
+| `video` | `theme/modules/prismic/PrismicEmbed/Components/Video.js` |
+| `link`  | `theme/modules/prismic/PrismicEmbed/Components/Link.js`  |
+| `rich`  | `theme/modules/prismic/PrismicEmbed/Components/Rich.js`  |
+
+As mentioned above, the fragment only exposes the minimum required fields for the PrismicEmbed component, if you would like to create your own custom embed component you can refer to all the available fields on the [`oEmbedContent`](https://gitlab.com/front-commerce/front-commerce-prismic/-/blob/main/prismic/server/modules/prismic/core/schema.gql) type.
+
+## Wysiwyg Embed Fields
+
+The [PrismicWysiwyg](/docs/advanced/theme/wysiwyg-platform.html#PrismicWysiwyg) type allows you to add embed fields to your Wysiwyg.
+
+### Prerequisites
+
+To get started with the PrismicWysiwyg, please ensure that you have applied the override to `theme/modules/WysiwygV2/getWysiwygComponent.js`, you can follow the documentation from [Wysiwyg](/docs/advanced/theme/wysiwyg.html), to understand how implement Wysiwyg.
```

Comment:
**suggestion:** as per my other comments, I'd suggest to redirect to a section form the Installation page that will contain the details below.\
This  section will only contain a single paragraph.